driver/src/test/java/org/neo4j/driver/integration/ConnectionHandlingIT.java

```diff
@@ -100,7 +102,7 @@ void createDriver()
         AuthToken auth = neo4j.authToken();
         RoutingSettings routingSettings = RoutingSettings.DEFAULT;
         RetrySettings retrySettings = RetrySettings.DEFAULT;
-        driver = driverFactory.newInstance( neo4j.uri(), auth, routingSettings, retrySettings, defaultConfig(), SecurityPlanImpl.insecure() );
+        driver = driverFactory.newInstance( neo4j.uri(), auth, routingSettings, retrySettings, Config.builder().withLogging( Logging.console( Level.FINE ) ).build(), SecurityPlanImpl.insecure() );
```

Comment:
just checking the logging is Intentional

Reply:
Sorry, this is a trace from debugging. I have updated it now.